Dockerfile

```
@@ -0,0 +1,26 @@
+FROM python:3 as base
+
+#ENV LIBRARY_PATH=/lib:/usr/lib
+
+RUN curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
+
+WORKDIR /todo_app
+
+COPY . /todo_app
+#copy dir
+ENV PATH="${PATH}:/root/.poetry/bin"
+RUN poetry install
+
+#EXPOSE 80
+#install poetry
+FROM base as development 
+ENTRYPOINT  poetry run flask run --host 0.0.0.0:80
+
+FROM base as production
+RUN poetry add gunicorn
+EXPOSE 80
+CMD ["gunicorn"  , "-b", "0.0.0.0:80", "todo_app.app:create_app()"]
```

I still see issues when I try to run this container:
>"gunicorn": executable file not found in $PATH: unknown.

Instead we should be running this with the below so that it knows where to find the dependency:
```suggestion
CMD ["poetry", "run", "gunicorn"  , "-b", "0.0.0.0:80", "todo_app.app:create_app()"]
```